scripts/query_collection.R

```diff
@@ -500,7 +500,9 @@ for (page in 1:last) {
            cat_sd = case_when(str_detect(topics, "category-software-dev") ~ "Software Development",
                               !str_detect(topics, "category-software-dev") ~ NA_character_),
            cat_bp = case_when(str_detect(topics, "category-best-practices") ~ "Best Practices",
-                              !str_detect(topics, "category-best-practices") ~ NA_character_)
+                              !str_detect(topics, "category-best-practices") ~ NA_character_),
```

Comment:
can you also please add how these are handled in the readme?

- that it is detecting xyz words from the topics description etc. 

I wonder if using tags would be any easier? (not suggesting we make that change necessarily though) just a thought